alerter/src/main/java/com/tapadoo/alerter/OnHideAlertListener.java

```diff
@@ -0,0 +1,15 @@
+package com.tapadoo.alerter;
+
+/**
+ * On Hide Alert Listener
+ *
+ * @author edgeorge
+ * @since 20/02/2017.
+ */
+
+public interface OnHideAlertListener {
+    /**
+     * Called when the Alert is hidden
+     */
+    void onHide();
```

Comment:
Good point - I think they'd be functionally the same thing, as this is executed when the Alert view is removed form the parent, either after the show duration has expired, or if the user touches the alert to dismiss it.